langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -99,6 +117,21 @@ public isolated function get(xml<ItemType> x, int i) returns ItemType = @java:Me
 } external;
 
 # Concatenates xml and string values.
+# 
+# ```ballerina
+# xml bookA = xml `<book>Ballerina</book>`;
+# xml bookB = xml `<book>Java</book>`;
+# xml:concat(bookA, bookB, xml `<book>Python</book>`) ⇒ <book>Ballerina</book><book>Java</book><book>Python</book>
+# 
+# bookA.concat(bookB) ⇒ <book>Ballerina</book><book>Java</book>
+# 
+# bookB.concat("Coding") ⇒ <book>Java</book>Coding
+# 
+# xml:concat("Hello", "World") ⇒ HelloWorld
+# 
+# xml[] subjects = [xml `<subject>English</subject>`, xml `<subject>Math</subject>`, xml `<subject>ICT</subject>`];
```

Comment:
Shall we add a similar example with string

Reply:
The `concat` operation on string type has been addressed at: #39036.